packages/example-app-base/src/components/MarkdownTable/MarkdownTable.module.scss

```diff
@@ -0,0 +1,59 @@
+@import '../../common/common';
```

Comment:
Would it be possible to use css-in-js here instead? This is a lot of scss to be adding in a brand new file when we're trying to move away from scss.

Reply:
I'll see if I can dedicate some time to do css-in-js instead.
Also tagging @Jahnp for visibility.

Reply:
For a new file, it's not terribly difficult. I've just recently been learning about it while working on the DetailsList examples in #7464. Here's an example of part of the conversion...though what you're doing is a bit more complicated and involves some things I'm not quite sure how to do yet (some guesses included below). @Vitalius1 would probably have a better idea of how to do complicated selectors with all the css-in-js component conversions he's been doing.

```ts
import { mergeStyleSets, DefaultPalette, DefaultFontStyles } from 'office-ui-fabric-react/lib/Styling';

// This generates a map of class names
const classNames1 = mergeStyleSets({
  table: {
    width: '100%',
    borderCollapse: 'collapse',
    // compiler complained about borderSpacing--maybe you meant cellSpacing?
    cellSpacing: 0,
    selectors: {
      '.ms-Image img': {
        maxWidth: '100%'
      }
    }
  },
  th: {
    fontWeight: 600,
    // Not sure if this is the correct way to get the colors...
    color: DefaultPalette.neutralPrimary,
    whiteSpace: 'nowrap',
    textAlign: 'left'

  },
  // ...

  // Not 100% sure how to handle nested selectors...my ideas:
  // - make a separate class and apply both it and the regular class in the JS file?
  // trInTh: { /*styles*/ }
});
// - make a second style object and build the nested selector from the generated class names?
// const finalClassNames = mergeStyleSets(classNames1, {
//   [`.${classNames1.th} .${classNames1.tr}`]: { /*styles */ }
// });
// - or some other way entirely?
```

Reply:
Looks good as far as I can tell. But I'd really like for someone else who's familiar with example-app-base to review the changes.